src/libs/actions/IOU.js

```diff
@@ -123,9 +123,7 @@ function buildOnyxDataForMoneyRequest(
             key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${chatReport.reportID}`,
             value: {
                 ...(isNewChatReport ? {[chatCreatedAction.reportActionID]: chatCreatedAction} : {}),
-                [reportPreviewAction.reportActionID]: {
-                    ...(isNewReportPreviewAction ? reportPreviewAction : {created: DateUtils.getDBTime()}),
-                },
+                [reportPreviewAction.reportActionID]: reportPreviewAction,
```

Comment:
It is already updated [here](https://github.com/Expensify/App/blob/e17a359f9eb0eadd4fa33bcb20ea6d98de9177ca/src/libs/actions/IOU.js#L189-L195)

Reply:
Thanks for detailing what you mean. It makes sense to update the optimistic data, because the reportPreviewAction changes always, no matter if it is newly created or updated. On the other hand, the success data changes only when the report preview action is created. 
Why do you think that `optimisticData` and `successData` should be consistent and have to look similar? Also, why would we trigger more Onyx updates for a simple syntactic sugar?
Let me know if I am missing something. Also, cc'ing @luacmartins and @mountiny for an opinion on this.

Reply:
I think it makes sense to always set the successData to null

Reply:
Fixed.